pages/record/_.vue

```diff
@@ -317,6 +326,8 @@
         }
       });
 
+      this.canonicalURL = window.location.href;
```

Comment:
This will be too lenient for when a user adds arbitrary stuff to the URL, like for example:
http://localhost:3000/en/record/2024909/photography_ProvidedCHO_United_Archives_02035211_jpg?param_that_does=nothing&another=one#user-supplied-tag
For now remove any params and anchor tags.